src/store/mutations.ts

```diff
@@ -210,8 +210,13 @@ export const mutateIsLoadingSubmission = (state: StateIF, isLoadingSubmission: b
 }
 
 export const mutateNameRequest = (state: StateIF, nr: NameRequestI) => {
+  // store NR data
   state.stateModel.newRequestModel.nr = nr
-  // TODO: have 1 mutation for nr / applicant info and put these there
+
+  // store Priority Code
+  mutatePriorityRequest(state, (nr?.priorityCd === PriorityCode.Y))
```

Comment:
This was a bug where the Priority flag was not retained when retrying a payment (for display on the fee summary; I think the payment object itself was OK).

Reply:
👍 great!